### PR TITLE
hotifx post

### PR DESCRIPTION
**Pull request by Poolitzer**

### CODE REVIEW

**Comment by TheAssassin** [comment]
Your post doesn't mention YouTube prominently. Please fix that.

**Comment by Poolitzer** [comment]
Not sure I get where you want me to put Youtube.

Here?
image

Here?
image

Or here?
image


Or somewhere even more prominently, like the title?

**Comment by TheAssassin** [comment]
Sorry, I meant to put it more prominently as in "I think it should be put in the title". You also should mention in the article's short text. Just noticed that your text lacks the separator, there should be one to make the list of posts more compact. That's where mentioning YT more prominently gets relevant.

**Comment by Poolitzer** [comment]
ohhh, I totally missed that, you are right. I will figure something out.

**Comment by Poolitzer** [comment]
how is that?

File: _posts/2020-01-26-newpipe-0.18.2-released.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "NewPipe 0.18.2 released: Hotfix for the YouTube-Extractor"
```
Comment (TheAssassin) [inline _posts/2020-01-26-newpipe-0.18.2-released.md]: That dash shouldn't be there, gramatically speaking.

File: _posts/2020-01-26-newpipe-0.18.2-released.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+short: "NewPipe 0.18.1 released"
```
Comment (TheAssassin) [inline _posts/2020-01-26-newpipe-0.18.2-released.md]: The version is `.2`, you forgot to update this line.

File: _posts/2020-01-26-newpipe-0.18.2-released.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+8 days after the last release and the next one is already out: A hotifx for the Extractor! A change on YouTubes side made it necessary.
+
+## Hotfix
+
+A hotfix release regarding the Extractor usually means that all the older versions of NewsPipe are likely to fail to play YouTube videos - this release is no difference. If you see any Guru Mediation error in the app, please update to this version before you create an issue (and/or verify that you actually have `0.18.2+` installed via hitting the `About` section in the left menu).
```
Comment (TheAssassin) [inline _posts/2020-01-26-newpipe-0.18.2-released.md]: The excerpt shouldn't include any heading. Also it's not `NewsPipe`, it's `NewPipe`.

File: _posts/2020-01-26-newpipe-0.18.2-released.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+<!-- more -->
+
+YouTube changed how they handled json. The actual fix was implemented by [@TobiGr](https://github.com/TobiGr), with the help of [@theScrabi](https://github.com/theScrabi) and (arguably even more helping) [@amuc](https://github.com/amuc). It was based on [@dstftw](https://github.com/dstftw) implementation in [youtube-dl](https://github.com/ytdl-org/youtube-dl). A big thanks goes to everyone involved.
```
Comment (TheAssassin) [inline _posts/2020-01-26-newpipe-0.18.2-released.md]: Should be `[@dstftw](...)'s`.

File: _posts/2020-01-26-newpipe-0.18.2-released.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+
+## Where to get the latest version
+
+NewPipe notifies you about new versions, you can download them when you press the notification. An alternative is the [Github release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), it notifies you as well about an update for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/) - please keep in mind that it can take F-Droid a while to update their repository.
```
Comment (TheAssassin) [inline _posts/2020-01-26-newpipe-0.18.2-released.md]: Can you really "press" notifications? You can "click on" them perhaps.

Comment (Poolitzer) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: its a mobile, I aint clicking there

File: _posts/2020-01-26-newpipe-0.18.2-released.md (hunk 3 above)
Comment (Poolitzer) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: I liked it as a separator from the headline, and also to state what the paragraph is about.

Its not NewsPipe though, correct.

File: _posts/2020-01-26-newpipe-0.18.2-released.md (hunk 4 above)
Comment (Poolitzer) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: yep

File: _posts/2020-01-26-newpipe-0.18.2-released.md (hunk 2 above)
Comment (Poolitzer) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: I did

File: _posts/2020-01-26-newpipe-0.18.2-released.md (hunk 1 above)
Comment (Poolitzer) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: me grammar great.

**Comment by Poolitzer** [comment]
okay, I changed the minor issues, now we just need to argue about the two open conversations.

File: _posts/2020-01-26-newpipe-0.18.2-released.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+title: "NewPipe 0.18.2 released: Hotfix for the YouTube Extractor"
+short: "NewPipe 0.18.2 released"
+date: 2020-01-26 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+8 days after the last release and the next one is already out: A hotifx for the Extractor! A change on YouTubes side made it necessary.
```
Comment (TobiGr) [inline _posts/2020-01-26-newpipe-0.18.2-released.md]:
```suggestion
8 days after the last release and the next one is already out: A hotifx for the Extractor! A change on YouTube's side made it necessary.
```

Comment (TheAssassin) [inline _posts/2020-01-26-newpipe-0.18.2-released.md]:
```suggestion
8 days after the last release and the next one is already out: A hotfix for the Extractor! A change on YouTube's side made it necessary.
```

Comment (TheAssassin) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: @Poolitzer hasn't allowed us to edit their branch. Took me 2 minutes to figure out why this is shown:

screenshot_2020-01-26_12-46-01

Please commit this last change and we can merge and publish your post.

Comment (Poolitzer) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: But I ticked the box :(

Comment (TobiGr) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: @TheAssassin I'll merge and push a commit to master fixing the typos.

Comment (Poolitzer) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: Fixed them already

Comment (TobiGr) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: Oh. GitHub was a bit slow and didn't show me the new commit.
Thanks @Poolitzer!

Comment (Poolitzer) [reply _posts/2020-01-26-newpipe-0.18.2-released.md]: You are welcome

**Pull request merged**